valet/pathproc.go

```diff
@@ -70,7 +70,14 @@ func ProcessFiles(cancelCtx context.Context, params ProcessParams) {
 		os.Exit(1)
 	}
 
```

Comment:
I haven't gone through and really understood the flow of things, but would DoProcessFiles() not benefit from also responding to the cancelled context? With the PR as it is, do you see the final summary messages from DoProcessFiles() on ctrl-c?

Reply:
You're right. I should have removed that `os.Exit()` in the goroutine and moved the remaining `os.Exit()` until after the remaining work has come thru the channel.